sources/packages/backend/apps/db-migrations/src/sql/Queue/Add-ecert-cancellation-response-integration.sql

```diff
@@ -0,0 +1,19 @@
+-- The scheduler is set to run at 5 p.m. UTC which is 9 a.m. PST every day.
+INSERT INTO
+    sims.queue_configurations(queue_name, queue_configuration, queue_settings)
+VALUES
+    (
+        'e-cert-cancellation-response-integration',
+        '{
+            "cron": "0 17 * * *",
+            "retry": 3,
+            "cleanUpPeriod": 2592000000,
+            "retryInterval": 180000,
+            "dashboardReadonly": false
+        }' :: json,
```

Comment:
I saw that we do a cast to JSON sometimes and sometimes don't, which means it is not required.
If the idea is to be explicit, maybe we should be specific and use the JSONB instead (if it is a valid cast).
Not a blocker.